### ISSUE TITLE
Welcome to join Open AI Robot GitHub Organisation

### ISSUE THREAD

**Issue by unbug** [unbug/Anki-Vector-AI#1]
![logo](https://avatars3.githubusercontent.com/u/49134548?s=400&u=d322d307dee6707bbd2ca97c30cfd35b62595cd8&v=4)

https://github.com/open-ai-robot

Let's be together to DIY real-world open-source AI-driven robot.

If you're interesting, please leave a comment, I'll invite you later :)

**Comment by ba2512005** [comment]
Hey Unbug! 

Looks like Anki got bought out by digital dream labs and they are now making Vector subscription based. It seems like they are offering an Escape pod service where you can host your own vector servers away from the dependency of digital dream labs. Have you looked much into this acquisition? What are your thoughts on how we should approach Vector dev going forward?

Regards,
Mo

**Comment by unbug** [comment]
@ba2512005 I won't pay for such retired service, deploy an AI service is easy like piece of cake on Google cloud and Microsoft Cloud.

**Comment by ba2512005** [comment]
@unbug  yeah I agree! I don't want to do it either however I'm wondering how we would integrate the updates from the service to our own hosted environment. Maybe we can use my friends bot who has the subscription and find a way to get the updated program. 

I was also thinking of integrating the object_detection subroutine with alexa so I can call it via an alexa skill and have it trigger a lambda function or something like that to connect to the robot and send it commands. 
I imagine the prompt would be like "Alexa, Ask vector to detect objects" or "Alexa, Ask vector what he's looking at". And have a bunch of prompts for different subroutines. This would enhance vector natively so you won't need a computer to push code to him if you want him doing something.

Have you explored something like this before? What has been your experience?

Regards,
Mo

**Comment by unbug** [comment]
@ba2512005 The Alexa has been buggy on Vactor，for example, if you set an alarm on Alexa, you won't be able to dismiss it from Vector. And most of the Alexa feature can't find a way to escape.

**Comment by ba2512005** [comment]
@unbug Yeah Alexa has been limited on him. I saw a video of something like this though which got me interested in doing it myself.

https://www.youtube.com/watch?v=Z_OmWVP-4-w&feature=youtu.be&ab_channel=AndrewHenley
"Alexa, trigger Vector to_____"

How do you imagine this person managed to get it working? Seems like a cool way to trigger custom actions. I was thinking we can build a library of custom actions we can call from alexa with different utterances. Open it up to the community so they can start adding to the library and link their alexa skill to vector. Do you know anyone else who programs on this robot?

**Comment by unbug** [comment]
@ba2512005 that's easy, log in your Alexa account on Amazon, you should find the docs to do so, you can also find some clue on the Anki forum

**Comment by ba2512005** [comment]
I've been reading into it however the Alexa skills developer console doesn't have debugging available for me to figure out why the skill isn't working. I also have to convert the authentication in object_detection.py to use Oauth with Google cloud vision api because the alexa skills developer console doesn't support custom environment variables.
